dependencies.yml

```diff
@@ -408,7 +406,7 @@ org.slf4j:
 
 org.springframework.boot:
   spring-boot-starter:
-    version: &SPRING_BOOT_VERSION '2.1.8.RELEASE'
+    version: &SPRING_BOOT_VERSION '2.1.9.RELEASE'
```

Comment:
We have decided to update to 2.1.9 because 2.2.0 does not support Tomcat 8.5. So this release(Armeria 0.95.0) will be the last release which supports Tomcat 8.5.

Reply:
/cc @adriancole

Reply:
cool..  thanks for update. also the things available for performance in boot 2.2 are actually spring related and may imply some refactoring as options were added to the `@Configuration` annotation. It may not actually require refactoring, but anyway.